common/perception_utils/include/perception_utils/perception_utils.hpp

```diff
@@ -54,6 +56,48 @@ namespace
     return boost::none;
   }
 }
+
+struct MinMaxPosition
+{
+  double max_x;
+  double min_x;
+  double max_y;
+  double min_y;
+};
+
+inline MinMaxPosition getMinMax2d(const tier4_autoware_utils::Polygon2d & polygon)
+{
+  auto init_point = polygon.outer().at(0);
+  double max_x = init_point.x();
+  double min_x = init_point.x();
+  double max_y = init_point.y();
+  double min_y = init_point.y();
+  for (const auto & point : polygon.outer()) {
+    if (point.x() > max_x) max_x = point.x();
+    if (min_x > point.x()) min_x = point.x();
```

Comment:
[IMO] use "else if" instead of "if"

Reply:
@scepter914 
Thank you for your comment! I removed this function in https://github.com/autowarefoundation/autoware.universe/commit/e9f857f42cbe0b91dc5e8562772d06c04dd0e234